external-builds/jax/README.md

```diff
@@ -0,0 +1,117 @@
+- Optional patches to be applied on top of a git checkout
+
+See the following table for how each version is supported:
+
+| JAX version   | Linux                                                                                                                                                                                                                                  | Windows           |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| master        | ✅ Supported<br><ul><li>[ROCm/rocm-jax `master` branch](https://github.com/ROCm/rocm-jax/tree/master)</li></ul>                                                                                                                        | ❌ Not supported  |
+| 0.7.1         | ✅ Supported<br><ul><li>[ROCm/rocm-jax `rocm-jaxlib-v0.7.1` branch](https://github.com/ROCm/rocm-jax/tree/rocm-jaxlib-v0.7.1)</li></ul>                                                                                                | ❌ Not supported  |
+
+
+
+## Build instructions
+
+For the most up-to-date and detailed build steps, please refer to the [Quick Build instructions in the ROCm JAX repository](https://github.com/ROCm/rocm-jax/tree/master?tab=readme-ov-file#quickbuild). Follow those steps to ensure a successful build process.
+
+## Build jax_rocmX_plugin and jax_rocmX_pjrt wheels instructions
+
+To build `jax_rocmX_plugin` and `jax_rocmX_pjrt` wheels, please follow the official instructions provided in the ROCm JAX repository. The most accurate and up-to-date build steps are documented at [BUILDING.md#building](https://github.com/ROCm/rocm-jax/blob/master/BUILDING.md#building). Refer to this guide for environment setup, build commands, and troubleshooting tips.
+
+
+## Developer Setup
+
+For JAX development setup and the latest instructions, please refer to the [ROCm JAX DEVSETUP guide](https://github.com/ROCm/rocm-jax/blob/master/DEVSETUP.md). If you want to run JAX locally, follow the steps outlined in that document for environment preparation and build commands.
+
+## Running/testing JAX
+
+After building the wheels, you should verify their functionality by running the recommended tests. Please follow the instructions provided in the [ROCm JAX BUILDING.md - Running Tests section](https://github.com/ROCm/rocm-jax/blob/master/BUILDING.md#3-running-tests) to ensure your build passes all required checks. This guide covers environment setup, test commands, and troubleshooting tips for validating your wheel builds.
+
+## Nightly releases
+
+### Gating releases with JAX tests
+
+With passing builds, we upload `jaxlib`, `jax_pjrt`, and `jax_plugin` wheels to subfolders of the "v2-staging" directory in the nightly release S3 bucket with a public URL at https://rocm.nightlies.amd.com/v2-staging/
```

Comment:
These are not the actual package names. Except for `jaxlib`, which is correct, theser have `rocm7` in the package name:

* `jax-rocm7-plugin`
* `jax-rocm7-pjrt`